test/modules/validation-resources/main.tf

```diff
@@ -0,0 +1,112 @@
+resource "tls_private_key" "ssh_keypair" {
+  algorithm   = "RSA"
+}
+
+// AWS
+
+data "aws_ami" "ubuntu" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-*"]
+  }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  owners = ["099720109477"] # Canonical
```

Comment:
Oh you mean Canonical as in the Ubuntu people...not that this ID is canonical...or am I the only one who would misunderstand this lol

Maybe `Canonical (Ubuntu) owner ID` or something? Or is it an account ID...you get what I mean...

Reply:
Copy pasted from Google's docs.  Probably only you though 😂